plugins/profefe.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.9.0
+  homepage: https://github.com/profefe/kube-profefe
+  shortDescription: Gather and manage pprof profiles from running pods
+  description: |
+    This plugin helps you do manage pprof profiles via Profefe. It helps you do
+    grab profiles from running pod. Load profile you have locally in Profefe
+    and inspect the profiles you have stored.
```

Comment:
For this plugin I think it would make sense to also link to the profefe homepage, because the referenced homepage https://github.com/profefe/kube-profefe is not the most natural site to get more information. So maybe include something like

> For more information about Profefe, visit https://profefe.dev